packages/iterator-utilities/package.json

```
@@ -687,7 +687,7 @@
 		}
 	},
 	"sideEffects": [
-		"./dist/esm/chunk-G5GHKT7C.mjs",
+		"./dist/esm/chunk-PAWJFY3S.mjs",
```

```suggestion
		"./dist/esm/chunk-G5GHKT7C.mjs",
```

This needs to be reverted otherwise changelog generation will put the commit to discord-utilities on iterator-utilities as well